views/about.lavender

```diff
@@ -20,22 +20,25 @@ block content
         .col-md-4.col-md-offset-2
           h1 Lavender
           p
-            | Lavender was formed out of a need for simplicity and ease. The syntax 
+            | Lavender was formed in the pursuit of simplicity and ease. The syntax 
             | is faster and easier to
-            | type, but functions very similar to Jade. If you feel like it, why not
-            a(href="https://github.com/golavender/lavender", target="_blank") fork us
-            | and help us make a difference.
+            | type and works like Jade. If you feel like it, why not
+            a(href="https://github.com/golavender/lavender", target="_blank") fork it
+            | and help us make a difference?
   .container
     .row
       .col-md-8.col-md-offset-2.about.img-rounded
         .col-md-5
           img.img-responsive.img-thumbnail.center-block(src="/img/daft-labs-logo.png")           
         .col-md-3.col-md-offset-2
-          h1 Daft Labs
+          h1 daft labs
           p
-            | Daft Labs was founded in 2013, slowly bringing together a team of hodge 
-            | podge people in both front end and back end development.  
-          a(href="http://www.daftlabs.com", target="_blank") Come Visit Us.
+            a(href='http://daftlabs.com/', target='_blank') daft labs
+            |  was founded in 2013, bringing together a hodge podge
+            | team of front and back end developers.  
+          a(href='http://www.daftlabs.com', target='_blank') Come Visit Us.
```

Comment:
There's a double link here (linking to daft labs at the beginning and end of the section.) Don't know if that's really necessary.